### PR TITLE
coin

### PR DESCRIPTION
**Pull request by osbentech**
This is my React-based capstone project named metrics webApp.
It fetches data from crypto API and displays it in the UI.
below are the functionalities I implemented:

   - When the page loads, the webapp shows the list of categories that could be filtered out by some parameter
   - When the user clicks (or taps) on a category item, the application navigates to the details page.
   - In the details page, the webapp retrieves data from the API to show detailed data in the category.
   - When the user clicks on the "Back" button (<), the user navigates to the home page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,12 +30,12 @@
...
 # JavaScript Capstone Project
 
-> This is a React Capstone projectnamed Metrics-webapp. Developed from wireframe and consumes the Air pollution API.
+> This is a React Capstone projectnamed Metrics-webapp. Developed from wireframe and consumes a CoinGecko API.
 
 <p align="right">(<a href="#top">back to top</a>)</p>
 
 ## Preview 📸
-<img width="800" alt="JS Capstone Preview Screenshot" src="./img/preview.png">
+<img width="800" alt="Preview Screenshot" src="./images/coinapp.png">
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly fix this or remove it as it is not displaying the image.

File: src/__test__/cryptohome.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import MyCryptoCurrencies from '../components/cryptohome';
+import store from '../redux/configureStore';
+
+it('Test MyCryptoCurrencies renders correctly', () => {
+  const tree = renderer
+    .create(
+      <Provider store={store}>
+        <MyCryptoCurrencies />
+      </Provider>,
+    )
```
Comment (sja-thedude) [inline src/__test__/cryptohome.test.js:12]: - [ ] Kindly make sure your tests are passed. Currently, they are failing.
Screenshot 2022-10-20 at 11 16 02 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @osbentech, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Professional Readme
✔️ Correct git-flow 

## Required 
- [ ] Check comments under the review....

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @osbentech 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**